database/src/main/postgres/flows/V1.7.4__flows.alter.ddl

```diff
@@ -13,6 +13,6 @@
  * limitations under the License.
  */
 
-ALTER COLUMN fk_primary_partitioning SET NOT NULL;
+ALTER TABLE flows.flows ALTER COLUMN fk_primary_partitioning SET NOT NULL;
```

Comment:
Already fixed in #167

Reply:
I know, but I would still like to keep it here because of easy testing - without this, my PR as a whole won't 'pass' and flyway will crash.